src/components/app-configure/general/AppBranding.vue

```diff
@@ -0,0 +1,192 @@
+<script lang="ts" setup>
+import { computed, reactive, type ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VDropdown from '@/components/lib/VDropdown/VDropdown.vue'
+import VFileUpload from '@/components/lib/VFileUpload/VFileUpload.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { availableThemes } from '@/utils/constants'
+
+type Theme = {
+  label: string
+  value: string
+}
+
+const store = useStore()
+
+type ThemeLogoOption = {
+  logo: File | undefined
+  horizontalLogo: File | undefined
+}
+
+type ThemeLogo = {
+  dark: ThemeLogoOption
+  light: ThemeLogoOption
+}
+
+const themeLogos: ThemeLogo = reactive({
+  dark: {
+    logo: undefined,
+    horizontalLogo: undefined,
+  },
+  light: {
+    logo: undefined,
+    horizontalLogo: undefined,
+  },
+})
+const selectedTheme: ComputedRef<Theme | undefined> = computed(() => {
+  return availableThemes.find(
+    (theme) => theme.value === store.getters.selectedTheme
+  )
+})
+
+function handleThemeChange(theme: Theme) {
+  store.commit('updateSelectedTheme', theme.value)
+}
+
+function handleFileChange(
+  mode: 'light' | 'dark',
+  logoType: 'logo' | 'horizontalLogo',
+  files: File[]
+) {
+  themeLogos[mode][logoType] = files[0]
+}
+
+function handleFileRemove(
+  mode: 'light' | 'dark',
+  logoType: 'logo' | 'horizontalLogo'
+) {
+  themeLogos[mode][logoType] = undefined
+}
+</script>
+
+<template>
+  <section name="app-branding">
+    <SettingCard class="app-branding-card">
+      <template #title>Branding</template>
+      <template #description
+        >Lorem ipsum dolor sit amet, consectetuer adipiscing elit, Lorem ipsum
+        dolor sit amet, consectetuer adipiscing elit.</template
+      >
+      <VStack direction="column" gap="2rem">
+        <VStack direction="column" gap="0.75rem" align="start">
+          <h4>Choose Theme</h4>
+          <VDropdown
+            :options="availableThemes"
+            display-field="label"
+            :model-value="selectedTheme"
+            class="theme-dropdown"
+            @update:model-value="handleThemeChange"
+          />
+        </VStack>
+        <VStack direction="column" gap="1.5rem">
+          <h3 class="text-uppercase">Upload Logo</h3>
+          <h4 class="text-uppercase font-700">Light Mode</h4>
+          <VStack gap="4rem" md-gap="2rem" wrap>
+            <VStack
+              direction="column"
+              gap="0.625rem"
+              class="file-upload-container"
+            >
+              <label>Logo Mark</label>
+              <VFileUpload
+                placeholder="Upload .png, .svg or .gif"
+                allowed-file-type=".png,.svg,.gif"
+                :value="themeLogos.light.logo?.name"
+                @file-change="handleFileChange('light', 'logo', $event)"
```

Comment:
Prefer naming events verb first.

```suggestion
                @change-file="handleFileChange('light', 'logo', $event)"
```